### PR TITLE
ref: Move column list out of Eventstore

### PR DESCRIPTION
**Pull request by lynnagara**
Since this column list is relevant for Snuba queries generally and not just for
Eventstore, let's move it out. This helps avoid the dependency of utils/snuba.py
on Eventstore.

### CODE REVIEW

**Review by fpacifici — APPROVED** [review]

File: src/sentry/snuba/events.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
+from __future__ import absolute_import
+
+from enum import Enum
+from collections import namedtuple
+
+Column = namedtuple("Column", "event_name transaction_name discover_name alias")
```
Comment (markstory) [inline src/sentry/snuba/events.py]: Long term we can likely remove the `transaction_name` as I don't think we have any present/known use cases for accessing the transactions dataset directly.

File: src/sentry/snuba/events.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    ISSUE = Column("issue", None, "group_id", "issue.id")
+    PROJECT_ID = Column("project_id", "project_id", "project_id", "project.id")
+    TIMESTAMP = Column("timestamp", "finish_ts", "timestamp", "timestamp")
+    TIME = Column("time", "bucketed_end", "time", "time")
+    CULPRIT = Column("culprit", None, "culprit", "culprit")
+    LOCATION = Column("location", None, "location", "location")
+    MESSAGE = Column("message", "transaction_name", "message", "message")
+    PLATFORM = Column("platform", "platform", "platform", "platform.name")
+    ENVIRONMENT = Column("environment", "environment", "environment", "environment")
+    RELEASE = Column("tags[sentry:release]", "release", "sentry:release", "release")
```
Comment (markstory) [inline src/sentry/snuba/events.py]: Isn't the discover name going to be `release`?

Comment (lynnagara) [reply src/sentry/snuba/events.py]: Thanks, made the same fix as https://github.com/getsentry/sentry/pull/15616

File: src/sentry/snuba/events.py (hunk 1 above)
Comment (lynnagara) [reply src/sentry/snuba/events.py]: Great

**Pull request merged**